packages/template-retail-react-app/app/utils/product-utils.test.js

```diff
@@ -49,3 +52,18 @@ test('getDisplayVariationValues', () => {
         Width: 'M'
     })
 })
+
+describe('getDisplayPrice', function () {
+    test('returns basePrice and discountPrice', () => {
+        const {basePrice, discountPrice} = getDisplayPrice(mockProductDetail)
+        expect(basePrice).toBe(mockedBasePrice)
+        expect(discountPrice).toBe(mockedDiscountPrice)
+    })
+
+    test('returns null if there is not discount promotion', () => {
+        delete mockProductDetail.productPromotions
```

Comment:
```suggestion
        const mock = {...mockProductDetail}
        delete mock.productPromotions
```
Avoid modifying the shared mock, otherwise other tests may not work as expected.

Reply:
it's not actually modifying the mock, just removing from the variable for the test. i created a new mock.